app/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,12 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" /> <!-- Used by sentry. -->
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="com.android.launcher.permission.INSTALL_SHORTCUT" />
-    <uses-feature android:name="android.hardware.touchscreen" android:required="false"/>
+
+    <!-- Required for voice control. It's only available on the Cube (so far) which is API 25+
+         so we use sdk-23 to prevent users from manually having to update the app.
+         https://developer.amazon.com/blogs/appstore/post/623addd5-6184-4e7f-a969-123a653cfb1a/implementing-voice-control-with-the-media-session-api-on-amazon-fire-tv -->
+    <uses-permission-sdk-23 android:name="com.amazon.permission.media.session.voicecommandcontrol" />
```

Comment:
Interesting - I guess if we start to ship on Android TV with #240 for non-Amazon use, this would be a permission that we'd have to make a flavor for (unless extraneous permissions have no effect).

Reply:
I think extraneous permissions have no effect but I don't know: added a note to the issue.